embeddings/evaluator/evaluator.py

```diff
@@ -9,8 +9,8 @@
 
 
 class Evaluator(ABC, Generic[Input, Output]):
-    def __init__(self) -> None:
-        pass
+    def __init__(self, return_input_data: bool = True) -> None:
+        self.return_input_data = return_input_data
```

Comment:
what is this parameter for? do you use it further in the code?

Reply:
It has been moved to MetricsEvaluator